### PR TITLE
ADD: session size configuration

### PR DESCRIPTION
**Pull request by CoffeeLatte007**
### Ⅰ. Describe what this PR did
1. Enable users to configure gloab session, branch session and write buffer size

2. For branch session, if it is too large, it will first try to compress lockkey, and if it is larger than the maximum, an exception will be thrown.

3. I want to do the compression of RPC, but the RPC protocol is not very friendly. If we want to do the body compression of RPC, we need to adjust the protocol design

### CODE REVIEW

File: server/src/main/java/com/alibaba/fescar/server/store/StoreConfig.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
+/**
```
Comment (xingfudeshi) [inline server/src/main/java/com/alibaba/fescar/server/store/StoreConfig.java]: copyright

File: server/src/main/java/com/alibaba/fescar/server/session/BranchSession.java [hunk 2]
```diff
@@ -35,9 +42,12 @@
  */
 public class BranchSession implements Lockable, Comparable<BranchSession>, SessionStorable {
 
-    public static final int DEFAULT_BRANCH_SESSION_BUFFER_SIZE = 2048;
+    private static final Logger LOGGER = LoggerFactory.getLogger(FileTransactionStoreManager.class);
```
Comment (xingfudeshi) [inline server/src/main/java/com/alibaba/fescar/server/session/BranchSession.java]: FileTransactionStoreManager.class?

**Comment by xingfudeshi** [comment]
i left some comments

**Comment by CoffeeLatte007** [comment]
> i left some comments

done

**Pull request merged**